source/mainnet/net/installation/downloads.rst

```diff
@@ -149,56 +209,107 @@ For the system requirements to run a node, see :ref:`System requirements to run
 
    When upgrading, you can only upgrade one minor version at a time, or from the last release of major version X to major version X+1. You cannot skip versions. For patches, you can skip versions e.g. X.X.0 to X.X.3, or `X.1.1` to `X.2.3`. To download previous node versions, see :ref:`Previous node versions<previous-downloads>`.
 
-Ubuntu v5.3.2
--------------
+.. _mainnet-node-downloads:
+
+.. dropdown:: Mainnet
+
+   .. dropdown:: Ubuntu |mainnet-node-version|
```

Comment:
Again, these version numbers kind of makes it look like it's e.g. Ubuntu that is version 5.3.2 
![image](https://github.com/Concordium/concordium.github.io/assets/76045015/f9373e66-3fdd-4553-9b6d-800dea66f5b6)

Reply:
but maybe it's just me

Reply:
I've been round and round with management, Ales, Kasper and everyone else. I had the version by Mainnet, by the Node headline and then Kasper didn't like that. But I was asked to put the version numbers next to it so it was clear what version the node was (or wallet or whatever) without the user needing to hover over it. In short, I've just changed this within the past two months and it is going to stay this way.

Reply:
Alright!